joint_position_trapezoid_controller:
  type: franka_panda_ip_controllers/JointPositionTrapezoidController
  arm_id: panda
  joint_names:
    - panda_joint1
    - panda_joint2
    - panda_joint3
    - panda_joint4
    - panda_joint5
    - panda_joint6
    - panda_joint7

